src/lib/components/image/_index.scss

```diff
@@ -0,0 +1,268 @@
+.vuiImage {
+  &__imageWrapper {
+    position: relative;
+    display: inline-block;
+    line-height: 0;
+  }
+
+  &__image {
+    display: block;
+    max-width: 100%;
+    height: auto;
+    transition: opacity $transitionSpeed;
+
+    &--loaded {
+      opacity: 1;
+    }
+
+    &--clickable {
+      cursor: pointer;
+    }
+  }
+
+  &__captionOverlay {
+    position: absolute;
+    bottom: 0;
+    left: 0;
+    right: 0;
+    padding: $sizeM $sizeM;
+    background: rgba(0, 0, 0, 0.7);
+    color: $colorEmptyShade !important;
+  }
+
+  &__previewOverlay {
+    position: absolute;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    gap: $sizeS;
+    background: rgba(0, 0, 0, 0.7);
+    backdrop-filter: blur(8px);
```

Comment:
This is cool, but I don't think it's needed. I'm a "less is more" kind of guy so I think we should remove it. 🙂